cmd/resolver.go

```diff
@@ -17,6 +18,8 @@ func init() {
 		"the path to workspace-folder")
 	resolveMtaCmd.Flags().StringVarP(&resolveModule, "module", "m", "",
 		"module-name")
+	resolveMtaCmd.Flags().StringVarP(&resolveEnvFileName, "envFileName", "e", "",
```

Comment:
I think the long parameter name should be `env` or `envFile` (it's descriptive enough)

Reply:
done

Reply:
I meant the parameter name (`"envFileName"`), not the variable name (which should start with `resolve` since it's used in the resolve command).

Reply:
OK, changed it now